### ISSUE TITLE
您好，我这是出现了什么问题，为何下载不了磁力链和bt种子文件

### ISSUE THREAD

**Issue by cy493069173** [aria2/aria2#994]
Exception: [NameResolver.cc:60] errorCode=19 Failed to resolve the hostname tracker.publicbt.com, cause: Name or service not known

**Comment by iLibra** [comment]
translation: Why I cannot download the BT and metalink files?

**Comment by cy493069173** [comment]
Yes, i download the magnetic chain file and the seed file will appear above error

**Comment by 4thleaf** [comment]
publicbt.com 这个域名没法解析。
- 也许你可以换个种子试试

**Comment by underlaw** [comment]
加上bt服务器地址
